flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/AbstractRestHandler.java

```diff
@@ -111,10 +112,13 @@ protected AbstractRestHandler(
 	private void processRestHandlerException(ChannelHandlerContext ctx, HttpRequest httpRequest, RestHandlerException rhe) {
 		log.error("Exception occurred in REST handler.", rhe);
 
+		String serverSideExceptionMesssage = "\n<Exception on server side:\n" +
```

Comment:
replace `\n` with system delimiter to be consistent with stack trace, remove leading/trailing newlines as this kind of formatting should be handled by whoever outputs the error.